Source/cmake/OptionsMSVC.cmake

```diff
@@ -63,3 +63,5 @@ cmake_path(REMOVE_FILENAME CMAKE_CXX_COMPILER OUTPUT_VARIABLE CLANG_CL_DIR)
 cmake_path(APPEND CLANG_CL_DIR "../lib/clang" ${CLANG_CL_MAJOR_VERSION} "lib/windows")
 find_library(CLANG_BUILTINS_LIBRARY clang_rt.builtins-x86_64 PATHS ${CLANG_CL_DIR} REQUIRED NO_DEFAULT_PATH)
 link_libraries(${CLANG_BUILTINS_LIBRARY})
+
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /STACK:8388608")
```

Comment:
Why does Windows port need such a big 8MB stack? Other port doesn't seem to use such a big stack.
https://github.com/WebKit/WebKit/blob/4d33c93bc32ae08a7958a1c59d59f13a06d18e39/Source/WTF/wtf/Threading.cpp#L108-L141

Reply:
We need a larger stack for the main thread - the `JSTests/stress/object-create-will-do-transition.js` test was overflowing the call stack. Mac and Linux both have 8MB stack size for the main thread, but with a 1MB stack size on Windows we're overflowing before we get close to the 1.5MB maxPerThreadStackUsage set for jsc stress tests.

We could update Threading.cpp#stackSize to keep a 1MB stack for other threads, we only need the increased stack size on the main thread.